src/components/erc721/collectibles/collectible_card.tsx

```diff
@@ -56,19 +51,66 @@ const Title = styled.h2`
 
 const defaultHandleClick = (e: any) => undefined;
 
-export const CollectibleCard: React.FC<Props> = (props: Props) => {
-    const { id, name, price, image, color, onClick, ...restProps } = props;
-    return (
-        <CollectibleCardWrapper
-            {...restProps}
-            id={id}
-            onClick={onClick || defaultHandleClick}
-            to={`${ERC721_APP_BASE_PATH}/collectible/${props.id}`}
-        >
-            <ImageWrapper color={color} image={image}>
-                <PriceBadge price={price} />
-            </ImageWrapper>
-            <Title>{name}</Title>
-        </CollectibleCardWrapper>
-    );
+interface OwnProps {
+    color: string;
+    id: string;
+    image: string;
+    name: string;
+    price: BigNumber | null;
+    onClick?: (e: any) => void;
+    currentOwner: string;
+    mustShowCollectibleOwnerBadge?: boolean;
```

Comment:
According to [this comment](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499190843) we will always have to show the badge, so the flag is not needed anymore.

Reply:
Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/5397939617dbd5a4cc53c523f75526f67c793a54)